core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXDatabaseServerV1Endpoint.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.distrox.api.v1.distrox.endpoint;
+
+import static com.sequenceiq.distrox.api.v1.distrox.doc.DistroXOpDescription.GET_DATABASE_SERVER_BY_CLUSTER_CRN;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.springframework.validation.annotation.Validated;
+
+import com.sequenceiq.cloudbreak.auth.altus.CrnResourceDescriptor;
+import com.sequenceiq.cloudbreak.jerseyclient.RetryAndMetrics;
+import com.sequenceiq.cloudbreak.validation.ValidCrn;
+import com.sequenceiq.distrox.api.v1.distrox.model.database.DistroXDatabaseServerResponse;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Validated
+@Path("/v1/distrox")
+@RetryAndMetrics
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(value = "/v1/distrox", protocols = "http,https", consumes = MediaType.APPLICATION_JSON)
+public interface DistroXDatabaseServerV1Endpoint {
```

Comment:
Why Is this in a different endpoint then other distrox calls

Reply:
There are multiple distrox* endpoints already and this database server request uses the cluster only for getting the database crn, nothing else. 
A similar endpoint created in datalake and these two should be the same.